app/gui/src/providers/rightPanel.ts

```diff
@@ -112,7 +111,6 @@ function useRightPanelTabs(
             return Err(getText('assetProjectExecutionsCalendar.teamPlanOnly'))
           return Ok()
         }),
-        hidden: computed(() => !toValue(enableScheduledExecution)),
```

Comment:
Are we sure we want to always show the tab? I thought we still want to hide it if the user is below Team plan.

Reply:
We should ideally show the tab with a message of only available in Team.
One of those why not upgrade messages :)